docs/Drawer.md

```diff
@@ -176,8 +178,60 @@ You can override the classes used by PX Blue by passing a `classes` prop. The Dr
+| chevron                   | Whether to have chevrons for all menu items                | `boolean`          | no       |                                                            |
+| collapseIcon              | Icon used to collapse drawer                               | `JSX.Element`      | no       | `expandIcon` rotated 180 degrees                           |
+| divider                   | Whether to show a line between all items                   | `boolean`          | no       | true                                                       |
+| expandIcon                | Icon used to expand drawer                                 | `JSX.Element`      | no       | `<ExpandLess />` at top-level, `<ArrowDropUp />` otherwise |
+| hidePadding               | Whether to hide the paddings reserved for menu item icons  | `boolean`          | no       |                                                            |
+| InfoListItemProps         | Used to override InfoListItem props set by the Drawer      | `InfoListItemProps`| no       |                                                            |
+| itemFontColor             | The color used for the item text                           | `string`           | no       | gray[500]                                                  |
+| itemIconColor             | The color used for the icon                                | `string`           | no       | gray[500]                                                  |
+| ripple                    | Whether to apply material ripple effect to items           | `boolean`          | no       | true                                                       |
+
+The following props control the NavGroup and thus only apply to `Drawer`, and `DrawerNavGroupProps` (so not `NavItem` or `NestedNavItem`):
+
+| Name                  | Description                                      | Type      | Required | Default                                                      |
+| --------------------- | ------------------------------------------------ | --------- | -------- | ------------------------------------------------------------ |
+| activeItem            | itemID for the 'active' item                     | `string`  | no       |                                                              |
+| nestedBackgroundColor | background color for nested menu items           | `string`  | no       | theme.palette.type === 'light' ? white[200] : black['A200'], |
+| nestedDivider         | Whether to show a line between nested menu items | `boolean` | no       | false                                                        |
+| titleColor            | Font color for group header                      | `string`  | no       | theme.palette.text.primary                                   |
+
+## DrawerNavItem
+
 #### NavItem Object
```

Comment:
Should NavItem come before the shared props? And doesn't NavItem also have the sharedProps?

Reply:
NavItem already has a sharedProps props.  I reordered the sharedProps and NavItem section.